gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/WsAndHttpChannelizerHandler.java

```diff
@@ -69,7 +69,6 @@ public void channelRead(final ChannelHandlerContext ctx, final Object obj) {
             //
             // IdleStateHandler -> HttpResponseEncoder -> HttpRequestDecoder ->
             //    WsAndHttpChannelizerHandler -> HttpObjectAggregator ->
-            //    WebSocketServerProtocolHandler ->
             //    WebSocketServerCompressionHandler -> WebSocketServerProtocolHandshakeHandler -> (more websockets)
```

Comment:
I think it'd be better to keep this step since it makes clear what line 80 is removing and provides context on what is being replaced by the HttpGremlinEndpointHandler

Reply:
That makes sense, I added it back